apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -93,7 +94,8 @@ export class JestPlugin implements IHeftPlugin {
       jestArgv._ = [...test.properties.findRelatedTests];
     }
 
-    const { results: jestResults } = await runCLI(jestArgv, [buildFolder]);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { globalConfig, results: jestResults } = await runCLI(jestArgv, [buildFolder]);
```

Comment:
?

Reply:
Jest's `Config.Argv` has a `[key: string]: string` escape hatch which effectively disables type checking.  And at runtime I unknown options are silently discarded.

So in the debugger, it's handy to have access to the `globalConfig` to verify that your options are actually being applied to the `globalConfig`.

Reply:
Can you leave a comment about that?

Reply:
Good idea

Reply:
Done